File: src/ios/Credentials.m
//
//
//

#import "Credentials.h"

@implementation Credentials
@synthesize appId, appKey;

//PLEASE ENTER YOUR APP ID INSIDE THE QUOTES BELOW
NSString* APP_ID = @"";
NSString* SKSAppKey = @"";
NSString* SKSServerHost = @"sslsandbox.nmdp.nuancemobility.net";
NSString* SKSServerPort = @"443";

-(NSString *) getAppId {
    return [NSString stringWithString:APP_ID];
};

-(NSString *) getServerUrl {
    return [NSString stringWithFormat:@"nmsps://%@@%@:%@", APP_ID, SKSServerHost, SKSServerPort];
};

@end

File: src/android/Credentials.java
package com.lingusocial.nuance;

import android.net.Uri;

public class Credentials {

	public static final String APP_KEY = "";

	public static final String APP_ID = "";

	public static final String SERVER_HOST = "sslsandbox.nmdp.nuancemobility.net";

	public static final String SERVER_PORT = "443";

	public static final Uri SERVER_URI = Uri.parse("nmsps://" + APP_ID + "@" + SERVER_HOST + ":" + SERVER_PORT);
}
